### DISCUSSION TITLE
How to set up TLS for RabbitMQ?

### DISCUSSION THREAD

**Discussion by nitin-kumar-mgr** [rabbitmq/rabbitmq-server#12233] [Other] [answered]
As i installed rabbitmq on different IP and created an inbound rule for the port to access from other pc. i want to set SSL to the IP.

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ is not installed "to an IP" and TLS is configured for listeners, not IP addresses.

Start with https://www.rabbitmq.com/docs/networking and https://www.rabbitmq.com/docs/ssl.
